frontend/app/src/components/asset-manager/ManagedAssetForm.vue

```diff
@@ -222,6 +218,9 @@ const saveAsset = async () => {
 
 const getUnderlyingTokenErrors = (underlyingTokens: any) => {
```

Comment:
not related to the PR but kind of know the proper formats of the underlying token errors based on the checks so maybe we can replace any?